### PR TITLE
HGT-132 Optional quit command

### PR DESCRIPTION
**Pull request by homapf**
Adding an option to remove the -quit argument when running Unity in batch mode.

Tested with
- Local Mac Runner (quit on and off)
- Ubuntu Cloud Runner (quit on and off)

Let's use this change in production and then do a PR in game-ci repository.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -213,6 +213,10 @@ inputs:
     description:
       'The path to mount the workspace inside the docker container. For windows, leave out the drive letter. For example
       c:/github/workspace should be defined as /github/workspace'
+  quit:
```
Comment (manelizzard) [inline action.yml]: The input here is `quit` but in the rest of the code is `shouldQuit`. Is this ok?

**Review by manelizzard — COMMENTED** [review]
Minor question

File: action.yml (hunk 1 above)
Comment (homapf) [reply action.yml]: Indeed, I think `shouldQuit` is better, changing.

**Review by manelizzard — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (manelizzard) [reply action.yml]: Thanks!

**Pull request merged**